main.cc

```diff
@@ -1029,9 +1029,6 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             // making compaction manager api available, after system keyspace has already been established.
             api::set_server_compaction_manager(ctx).get();
 
-            supervisor::notify("loading non-system sstables");
-            replica::distributed_loader::init_non_system_keyspaces(db, proxy, sys_ks).get();
-
             supervisor::notify("starting view update generator");
```

Comment:
Is it safe to move this after the components below?

Moving it after enable_features_on_startup is probably an improvement.

Reply:
I'm not 100% sure about view_update_generator. To be safe, I'll move it after init_non_system_keyspaces(). 

Other things seem safe to reorder.